docs/edd-request-form.html

```diff
@@ -73,65 +73,65 @@
         </div>
       </div> <!-- /nypl-column-three-quarters -->
     </div>
-    <form id="edd-request">
+    <form id="edd-request" class="nypl-electronic-delivery-form">
       <fieldset>
-      <div class="nypl-row">
+        <div class="nypl-row">
           <div class="nypl-column-half">
-
-              <h4>Contact Information</h4>
-              <div class="nypl-text-field" id="email-address">
-                <label for="required-field" id="email-address-label">Email Adress
-                  <span class="nypl-required-field">Required</span>
-                </label>
-                <input id="required-field" type="text" required="true" aria-labelledby="email-address-label email-address-status" aria-required="true" placeholder="" />
```

Comment:
Like we found with GALC, `required="true"` causes problems, so just use `aria-required`